### PR TITLE
More coverage work

### PR DESCRIPTION
**Pull request by andreas-kupries**
Ref #2307

This is all small stuff in low-level code, low-hanging fruit for unit tests, and unused code.

### CODE REVIEW

**Review by enrichman — DISMISSED** [review]

File: helpers/bytes/bytes_test.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+			/*K*/ {1 + 1024, "1.0 KiB"},
+			/*M*/ {1 + 1024*1024, "1.0 MiB"},
+			/*G*/ {1 + 1024*1024*1024, "1.0 GiB"},
+			/*T*/ {1 + 1024*1024*1024*1024, "1.0 TiB"},
+			/*P*/ {1 + 1024*1024*1024*1024*1024, "1.0 PiB"},
+			/*E*/ {1 + 1024*1024*1024*1024*1024*1024, "1.0 EiB"},
+		} {
+			r := bytes.ByteCountIEC(testcase.count)
+			Expect(r).To(Equal(testcase.result))
+		}
```
[the remark this answers is not in the document]
Comment (andreas-kupries) [reply helpers/bytes/bytes_test.go:37]: Thanks. That looks good. (Might also be usable for the command completion tests).

[the remark this answers is not in the document]
Comment (andreas-kupries) [reply helpers/bytes/bytes_test.go]: Resolved with commit 04f6140c6d06819d0ede7130340350f57baf0d8d

**Review by enrichman — CHANGES_REQUESTED** [review]
Just a couple of typos and a question regarding the ContainerState vars

File: helpers/kubernetes/tailer/container_state.go [hunk 2]
```diff
@@ -11,34 +11,10 @@
...
-	v1 "k8s.io/api/core/v1"
-)
-
 type ContainerState string
 
 const (
 	RUNNING    = "running"
 	WAITING    = "waiting"
 	TERMINATED = "terminated"
 )
```
[the remark this answers is not in the document]
Comment (andreas-kupries) [reply helpers/kubernetes/tailer/container_state.go:20]: Will check.

[the remark this answers is not in the document]
Comment (andreas-kupries) [reply helpers/kubernetes/tailer/container_state.go]: Indeed not used anywhere else.
The type is used. See setup.go and watcher.go of this package.

Commit bddd8922e12e9460d8817a4357804203c07f732f removes the strings

**Review by enrichman — APPROVED** [review]

**Pull request merged**